Makefile

```diff
@@ -329,9 +338,25 @@ verify-previous-version: ## Verifies that PREVIOUS_VERSION variable is set
 .PHONY: bundle-community-rh
 bundle-community-rh: bundle ## Update displayName field in the bundle's CSV
 	sed -r -i "s|displayName: Machine Deletion Remediation operator.*|displayName: Machine Deletion Remediation Operator - Community Edition|;" ${CSV}
+	$(MAKE) add-replaces-field
```

Comment:
I don't find this target?

Reply:
Doh, sorry, it's a leftover.
MDR only have 1 bundle (OKD) that needs the replaces field, so there's no need to specialize this part